perception/traffic_light_fine_detector/src/nodelet.cpp

```diff
@@ -0,0 +1,305 @@
+// Copyright 2023 Tier IV, Inc.
```

Comment:
```suggestion
// Copyright 2023 TIER IV, Inc.
```

Reply:
updated in commit 2606f617ff638fa0cd866f0f0c058e464ee049a9